lib/mix/lib/mix/tasks/help.ex

```diff
@@ -4,11 +4,12 @@ defmodule Mix.Tasks.Help do
   @shortdoc "Prints help information for tasks"
 
   @moduledoc """
-  Lists all tasks or prints the documentation for a given task.
+  Lists all aliases and tasks or prints the documentation for a given task.
 
   ## Arguments
 
-      mix help                  - prints all tasks and their shortdoc
+      mix help                  - prints all aliases, tasks and task shortdoc
```

Comment:
I think there is a typo here "prints all aliases, tasks and **their** shortdoc"

Reply:
I don't think we have shortdoc for aliases. shortdoc are marked with `@shortdoc`

Reply:
"and their summary" or "and their short description"?